dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
```

Comment:
Ah, redhat! Thank you.
And using docker on windows is certainly the right answer for windows; it's certainly too far from posix. But on gentoo etc., maybe we should do this, but maybe again we should detect and use yum instead. Ideally both.
But more to the point: what I hear is confirmation that dobi does not abstract installation of packages on the base platform, but helps to install packages within the container. If working on a redhat-based container is a goal, and it is a worthy goal, then I would contend that working naked on a redhat-based distro is also a goal. Doing this at the fabfile level becomes independently useful. (If we decide we can stick to an ubuntu container, then dobi abstracting package installation within the container is irrelevant, no? But I don't want to think like that.)
What may be true is that installing some packages at the dobi level before the fabfile runs might be more efficient, and that this part of the fabfile should end up being a noop. But that's a different matter.